packages/extension-typography/src/typography.ts

```diff
@@ -105,32 +129,97 @@ export const threeQuarters = textInputRule({
   replace: '¾',
 })
 
-export const Typography = Extension.create({
+export const Typography = Extension.create<TypographyOptions>({
   name: 'typography',
 
   addInputRules() {
```

Comment:
You should also define `addOptions()` with its default values here.

Reply:
If we keep the current approach where the options can only be set to `false`, I believe we don't need `addOptions`. Again, followed the exact same approach as `StarterKit` to keep things simple. But I can change to `boolean` and add `addOptions` with default set to `true`.